missions/W5/M1/taxi_rdd_analysis.py

```
@@ -0,0 +1,52 @@
+    row.fare_amount is not None and
+    row.trip_distance > 0 and 
+    row.fare_amount > 0 and
+    row.tpep_pickup_datetime.year == 2021
+)).cache()
+
+# 2. 데이터 파싱: (날짜, (거리, 요금, 1)) 형태로 변환
+def parse_row(row):
+    try:
+        date = row.tpep_pickup_datetime.date().isoformat()
+        return (date, (row.trip_distance, row.fare_amount, 1))  # (날짜, (총 거리, 총 요금, 트립 수))
+    except:
```

Ruff (0.8.2)

Do not use bare `except`

(E722)